rpc/cloudagent/compute/virtualmachine/moc_cloudagent_virtualmachine.proto

```diff
@@ -109,6 +109,11 @@ message AvailabilitySetReference {
 	string name = 2;
 }
 
+message AvailabilityZoneReference {
```

Comment:
what is this for? You shouldn't have any changes in this file.

Reply:
removed it after our discussion on virtual machine reference